src/Files.Uwp/BaseLayout.cs

```diff
@@ -367,7 +367,7 @@ public virtual void ResetItemOpacity()
                 return;
             }
 
-            foreach (var item in items.ToList()) // ToList() is necessary
```

Comment:
ToList() in _not_ necessary here :) Only useful when either:
- the list is modified inside the forearch
- the foreach contains an "await"